tests/sys/posixshm/posixshm_test.c

```diff
@@ -1679,14 +1679,14 @@ ATF_TC_BODY(largepage_mprotect, tc)
 		 * Reserve a contiguous region in the address space to avoid
 		 * spurious failures in the face of ASLR.
 		 */
-		addr = mmap(NULL, ps[i] * 2, PROT_NONE,
+		res = mmap(NULL, ps[i] * 2,
+		    PROT_NONE | PROT_MAX(PROT_READ | PROT_WRITE),
```

Comment:
Why set prot_max to anything other than PROT_NONE? This mapping is going to be replaced.

Reply:
I think this is because we use maxprot to produce capability permission bits and those are used later to authorise the MAP_FIXED replacement operation. Not sure whether this is the right model, but I thought it made sense at the time.

Reply:
The content of the mapping are replaced, the reservation remains.